data/stoa0022/stoa029/stoa0022.stoa029.opp-lat1.xml

```diff
@@ -40,33 +42,27 @@
       </publicationStmt>
       
       <sourceDesc>
-        <listBibl xml:lang="lat">
+        <listBibl>
           <biblStruct>
             <monogr>
+              <title xml:lang="lat">Sanctii Ambrosii Opera</title>
               <editor>
                 <persName>
-                  <name xml:lang="lat">Carolus Schenkl</name>
-                  <addName xml:lang="de">Karl Schenkl</addName>
+                  <name>Karl Schenkl</name>
                 </persName>
               </editor> 
-              <author ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022">Ambrosius</author>  
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa034">De Iacob</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa035">De Ioseph</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa019">De Patriarchis</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa029">De Fuga Saeculi</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa032">De Interpellatione Iob et David</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa015">De Apologia David</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa014">Apologia David Altera</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa025">De Helia et Ieiunio</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa038">De Nabuthae</title>
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0022.stoa044">De Tobia</title>
+              <author>Ambrose</author>  
               <imprint>
-                <publisher>Österreichische Akademie der Wissenschaften (ÖAW)</publisher>
-                <pubPlace>Wien, Österreich</pubPlace>
+                <publisher xml:lang="deu">Österreichische Akademie der Wissenschaften (ÖAW)</publisher>
+                <pubPlace xml:lang="deu">Wien, Österreich</pubPlace>
                 <date>1897</date>
               </imprint>
-              <biblScope unit="volume">32.2</biblScope>
+              <biblScope unit="volume">Pars Altera</biblScope>
```

Comment:
I've seen this elsewhere, and I wonder if it should be tagged Latin? I think we've got other texts where this is just changed to "2"? I'm not sure we kept this language in other works.
You probably want to keep as is for now but consider a global pass at some point.

Reply:
That's a good plan, I think I will keep working on files and then can go a global pass for consistency. Thanks so much for reviewing them all.